oceannavigator/oceannavigator.cfg

```diff
@@ -6,9 +6,8 @@ OVERLAY_KML_DIR = "./kml"
 DRIFTER_AGG_URL = "http://10.0.3.56:8080/thredds/dodsC/Drifter/aggregated.ncml"
 DRIFTER_URL = "http://10.0.3.56:8080/thredds/dodsC/Drifter/%s.nc"
 DRIFTER_CATALOG_URL = "http://10.0.3.56:8080/thredds/catalog/Drifter/catalog.xml"
-CLASS4_CATALOG_URL = "http://10.0.3.56:8080/thredds/catalog/Class4/catalog.xml"
-CLASS4_URL = "http://10.0.3.56:8080/thredds/dodsC/Class4/%s/%s.nc"
-CLASS4_FTP = "10.0.3.9"
+CLASS4_FNAME_PATTERN = "/data/class4/%s/%s.nc"
```

Comment:
Did a find+replace for this. A bit funky but as I said I don't want to radically change the overall architecture here.